aiogram/dispatcher/event/handler.py

```diff
@@ -59,11 +107,6 @@ class HandlerObject(CallableMixin):
     callback: HandlerType
     filters: Optional[List[FilterObject]] = None
 
-    def __post_init__(self) -> None:
-        super(HandlerObject, self).__post_init__()
-        if inspect.isclass(self.callback) and issubclass(self.callback, BaseHandler):  # type: ignore
-            self.awaitable = True
```

Comment:
How do you plans to detect awaitable callbacks?

Reply:
callback are naturally inspected in `CallableMixin`. I didn't actually change the behaviour of inspection, did I? :thinking:
This is check determined whether `BaseHandler` subclass was passed to `HandlerObject` and seemed redundant to me, since we can check callback once in `CallableMixin`'s `__post_init__`

Reply:
I need to check it